bt-target/client/main.lua

```diff
@@ -75,6 +80,8 @@ function playerTargetEnable()
 
                                     if GetEntityType(entity) == 0 or #(plyCoords - coords) > Models[_]["distance"] then
                                         success = false
+                                        targetActive = false
+                                        SendNUIMessage({response = "closeTarget"})
```

Comment:
This closes the whole menu, it's only needed for vehicle bones to stop them from conflicting with each other. This isn't needed for zones or models.

Reply:
if you release the ALT after valid target is triggered then you move your mouse away the "eye" remain still, this fix the problem.

https://gyazo.com/0991e506dd2c371c06ab5dc0954b4c63
The ALT was released after aiming to the box zone, i had to click ALT again to "remove the eye"

Reply:
In my tests this closes the menu when you aim away, it doesn't stay. The NUI message `closeTarget` sets the current label to an empty string an hides the `target-wrapper` div which contains everything. The code will trigger `leftTarget` instead just 1 tick later because `success` is being set to false and therefore breaking the loop. This was needed for the bones because there was an issue where conflicting bones caused the menu to be stuck, but it's not needed here.

Reply:
ok, just leave it like that so